config/prow/plugins.yaml

```diff
@@ -19,7 +19,26 @@ triggers:
   - containerd/containerd
   join_org_url: "https://github.com/containerd/project/blob/master/MAINTAINERS"
 - repos:
```

Comment:
@PatrickXYS Lets add a comment here explaining why the default is no longer "on" for new repos.

Can we also file an issue to track updating the new repository setup guide
https://github.com/kubeflow/community/blob/master/repository-setup.md

/cc @Bobgy

Reply:
I added comments to explain the reason, also, file issue here to keep track https://github.com/kubeflow/community/issues/450